kivy/data/style.kv

```diff
@@ -620,7 +620,6 @@
 
 <ActionPrevious>:
     size_hint_x: 1
-    minimum_width: layout.minimum_width + min(sp(100), title.width)
```

Comment:
Setting the `minimum_width` achieves nothing, it's counterproductive even:
the event gets triggered twice for some reason and it ends up at 56 instead
of the expected 156.  Also, the docs say these properties are supposed
to be read-only (see `BoxLayout`).